### PR TITLE
Make  the colors of the "Message tree states" match

### LINKED ISSUE #1801: Have the colors of the "Message tree states" match
The new pie charts for the stats look great, but as of right now, both "Message tree states" charts use the same color palette, but assign different colors to the same value. This makes things a little hard to compare - it's especially unfortunate that "Ready to Export" in English is colored gray, which makes it look like an unfavorable state. It looks like the colors are randomly assigned, but can you use the same seed or something like that for all of them?

### PR DESCRIPTION
**Pull request by GuilleHoardings**
Fixes #1801

### CODE REVIEW

File: website/src/components/Stats/Stats.components.tsx [hunk 1]
```diff
@@ -249,10 +267,30 @@ export const MessageTreeStateStatsStacked = ({ stat }: MessageTreeStateStatsProp
...
+
 export const Chart = ({ stat, titleFn, type: Component }: ChartProps) => {
-  const data = Object.keys(stat.stats);
+  let data = Object.keys(stat.stats);
   const { t } = useTranslation("stats");
   const color = useColorModeValue(colors.light.text, colors.dark.text);
+
+  if (stat.name === "message_trees_by_state") {
+    // Order data by interesting state
+    data = data.sort((a, b) => {
```
Comment (notmd) [inline website/src/components/Stats/Stats.components.tsx:287]: You dont need to reassign the `data` variable. `sort` mutate the original array

File: website/src/components/Stats/Stats.components.tsx [hunk 2]
```diff
@@ -173,6 +184,11 @@ export const MessageTreeStateStats = ({ stat, titleFn }: MessageTreeStateStatsPr
   const uniqueLangs = Array.from(new Set(stat.stats.map((s) => s.lang)));
   const [selectedLang, setSelectedLang] = useState("en");
   const stats = stat.stats.filter((item) => item.lang === selectedLang);
+  const statsOrdered = stats.sort((a, b) => {
```
Comment (notmd) [inline website/src/components/Stats/Stats.components.tsx:187]: same here, just chain the `sort`

File: website/src/components/Stats/Stats.components.tsx (hunk 1 above)
Comment (GuilleHoardings) [reply website/src/components/Stats/Stats.components.tsx]: I didn't know that, thank you. Fixed!

File: website/src/components/Stats/Stats.components.tsx (hunk 2 above)
Comment (GuilleHoardings) [reply website/src/components/Stats/Stats.components.tsx]: Also fixed.

File: website/src/components/Stats/Stats.components.tsx [hunk 3]
```diff
@@ -53,6 +53,17 @@ const interestingStates = [
   "halted_by_moderator",
 ];
 
+const stateColors = {
```
Comment (AbdBarho) [inline website/src/components/Stats/Stats.components.tsx:56]: could you rename to something like `STATE_COLORS`? since this is a constant.

Comment (GuilleHoardings) [reply website/src/components/Stats/Stats.components.tsx]: Done!

**Review by AbdBarho — APPROVED** [review]
Thanks

**Pull request merged**